src/eliater/discover_latent_nodes.py

```diff
@@ -14,19 +14,23 @@
 treatment's impact on the outcome. This focus results in more precise estimates with reduced
 variance and bias.
 
-Here is an example:
-
-.. todo::
-
-    Don't just give some random example. Motivate it. Explain the characteristics of the
-    example ADMG that are important. Explain what the algorithm does to it.
-
-.. todo::
-
-    Make sure to include correct imports such that this example can be directly
-    copy-pasted into a repl/jupyter
+Here is an example of an ADMG where X is the treatment and Y is the outcome is Y. This ADMG has
+only one causal path from X to Y which is X -> M1 -> M2 -> Y. The descendants of these variables
+that are ancestors of the outcome are R1, R2, and R3. The goal of this example is to identify these
+nuisance variables and mark them as latent.
```

Comment:
Why?

Reply:
One paragraph above this paragraph, I stated the reason for why we mark variables as latent:

"This process enables us to concentrate on the fundamental variables needed to estimate the
treatment's impact on the outcome. This focus results in more precise estimates with reduced
variance and bias."

Do you think I should add a description at the end of the example paragraph?

Reply:
I mean why are they marked as latent. What's the purpose of doing this? It doesn't accomplish the following:

>This process enables us to concentrate on the fundamental variables needed to estimate the
treatment's impact on the outcome. This focus results in more precise estimates with reduced
variance and bias.